src/resources/template.html

```diff
@@ -24,5 +24,6 @@
 		<%= htmlWebpackPlugin.options.ssr({
 			url: '/'
 		}) %>
+		<script>"undefined"==typeof fetch&&document.write('<script src="<%= htmlWebpackPlugin.files.publicPath + "polyfills.js" %>"><\/script>')</script>
```

Comment:
Lol okay. I was just going for perf since `!== void 0` is faster than `typeof === 'undefined'`